tests/bootstrap.php

```diff
@@ -2,11 +2,85 @@
 
 declare( strict_types=1 );
 
-require_once __DIR__ . '/../vendor/autoload.php';
-require_once __DIR__ . '/../vendor/php-stubs/wordpress-stubs/wordpress-stubs.php';
-require_once __DIR__ . '/../vendor/php-stubs/woocommerce-stubs/woocommerce-stubs.php';
-require_once __DIR__ . '/../vendor/php-stubs/woocommerce-stubs/woocommerce-packages-stubs.php';
-require_once __DIR__ . '/../deps/scoper-autoload.php';
-require_once __DIR__ . '/../constants.php';
-// To import constants.
-require_once __DIR__ . '/../vendor/szepeviktor/phpstan-wordpress/bootstrap.php';
+$args = $_SERVER['argv'] ?? [];
+
+$currentTestsuite = null;
+$testsuiteIndex   = array_search( '--testsuite', $args, true );
+if ( $testsuiteIndex !== false && isset( $args[ $testsuiteIndex + 1 ] ) ) {
+	$currentTestsuite = $args[ $testsuiteIndex + 1 ];
+}
+
+if ( $currentTestsuite === null ) {
+	foreach ( $args as $arg ) {
+		if ( strpos( $arg, '--testsuite=' ) === 0 ) {
+			$currentTestsuite = substr( $arg, strlen( '--testsuite=' ) );
+
+			break;
+		}
+	}
+}
+
+if ( $currentTestsuite === null ) {
+	die( 'Testsuite detection failed.' );
+}
+
+if ( $currentTestsuite === 'unit' ) {
```

Comment:
currentTestsuite může být enum, protože má 2 varianty, když tam přidám nesmysl tak script projde ale nic nespustí.

Reply:
to sice není enum, ale array, nicméně ok.